### PR TITLE
Handle overrides

### PR DESCRIPTION
**Pull request by sigurdm**
Handle the case where a dependency is overridden.

Also print the source name for packages that are not from a hosted source.

### CODE REVIEW

**Comment by sigurdm** [comment]
travis???

File: lib/src/command/outdated.dart [hunk 1]
```diff
@@ -65,11 +64,9 @@ class OutdatedCommand extends PubCommand {
     entrypoint.assertUpToDate();
 
     final includeDevDependencies = argResults['dev-dependencies'];
-
     final upgradePubspec = includeDevDependencies
-        ? entrypoint.root.pubspec
+        ? _stripOverrides(entrypoint.root.pubspec)
         : _stripDevDependencies(entrypoint.root.pubspec);
```
Comment (jonasfj) [inline lib/src/command/outdated.dart]: Need to rename this `stripOverridesAndDevDeps` or we do:
`_stripDevDependencies(_stripOverrides(entrypoint.root.pubspec))`.

And fix the todo in `_stripDevDependencies` by preserving `dependency_overrides`.

**Comment by sigurdm** [comment]
We decided to not handle overrides in first release.

I'll close this for now.

File: lib/src/command/outdated.dart [hunk 2]
```diff
@@ -188,7 +181,16 @@ Pubspec _stripDevDependencies(Pubspec original) {
     sdkConstraints: original.sdkConstraints,
     dependencies: original.dependencies.values,
     devDependencies: [], // explicitly give empty list, to prevent lazy parsing
-    // TODO(sigurdm): consider dependency overrides.
```
Comment (jonasfj) [inline lib/src/command/outdated.dart]: preserve overrides

**Pull request closed without merging**